.github/workflows/pull-release.yml

```diff
@@ -0,0 +1,137 @@
+name: get latest helm charts
+on:
+  schedule:
+    - cron: '00 06 * * 5' #every friday at 11:30 AM IST/ 6 AM UST
+
+
+jobs:
+  get-latest-releases:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Get latest charts for kafka
+        id: kafka
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/kafka
+
+      - name: Get latest charts for zookeeper
+        id: zookeeper
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/zookeeper
+
+      - name: Get latest charts for pinot
+        id: pinot
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/pinot
+
+      - name: Get latest charts for schema-registry 
+        id: schema-registry
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/schema-registry
+
+      - name: Get latest charts for mongodb
+        id: mongodb
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/mongodb
+
+      - name: Get latest charts for oc-collector
+        id: opencensus-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/opencensus-service
+
+      - name: Get latest charts for ingester
+        id: hypertrace-ingester
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-ingester
+   
+      - name: Get latest charts for ui
+        id: hypertrace-ui
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-ui
+
+      - name: Get latest charts for graphql
+        id: hypertrace-graphql
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-graphql
+
+      - name: Get latest charts for attribute-service
+        id: attribute-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/attribute-service
+
+      - name: Get latest charts for gateway-service
+        id: gateway-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/gateway-service
+
+      - name: Get latest charts for query-service
+        id: query-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/query-service
+
+      - name: Get latest charts for entity-service
+        id: entity-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/entity-service
+
+      - uses: actions/checkout@v2
+
+      - name: Update data services charts
+        run: ./.github/workflows/create-charts.sh data-services
+        env: 
+          kafka_version: ${{ steps.kafka.outputs.tag }}
```

Comment:
How does fetching of version work here?

Reply:
`oprypin/find-latest-tag@v1` this action pull latest tags from repositories and output of that step can be accessed with `steps.kafka.outputs.tag`

Reply:
so this version comes from previous steps.

Reply:
ok, so the syntax is `steps.<action_id>.outputs.tag`

Reply:
yup